cpp/src/aztec3/circuits/abis/membership_witness.hpp

```diff
@@ -15,7 +17,7 @@ template <typename NCT, unsigned int N> struct MembershipWitness {
     typedef typename NCT::boolean boolean;
 
     uint32 leaf_index;
-    std::array<fr, N> sibling_path = { 0 };
+    std::array<fr, N> sibling_path = zero_array<fr, N>();
```

Comment:
Similarly, if fr is zero-initialized by its default constructor this is not needed